libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3158,6 +3158,14 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 
 			annotation.isPure = isPure;
 		}
+	if (
+		VariableDeclaration const* varDecl = dynamic_cast<decltype(varDecl)>(annotation.referencedDeclaration);
```

Comment:
```suggestion
		auto const* variableDeclaration = dynamic_cast<VariableDeclaration const*>(annotation.referencedDeclaration);
```

Reply:
Done. I'm keeping variable name though since `varDecl` seems to be the name used for `VariableDeclaration` elsewhere.

Reply:
I think some of the older code is overusing these shortened names. Personally I'd use full words but fair enough, we do have these shortened names in some places.